menuconfig.py

```diff
@@ -291,6 +292,10 @@ def style_from_string(sstr):
 
     return style
 
+def style_set(sstr):
```

Comment:
Think I'd prefer for there to just be a single mechanism, at least until we run into any limitations with it.

To set the style programmatically, you could always just set `os.environ["MENUCONFIG_STYLE"]` in the code.